ONSdigital/florence
# Feature/datasets in collections

### What

Update the collections screen to support CMD datasets being in a collection:

- Show in progress, awating review or reviewed datasets and dataset versions
- Update the 'Approve' and 'Delete' collection buttons logic to recognise datasets and dataset version being in the collection.
- Clean up some link styling broken by default `:visited` styling being on <a> tags on CMD develop.

**Note**: This does not add the functionality to take the datasets through the review process, but they should display correctly if they have the correct information in the collection's JSON file.

### How to review

1) You need Zebedee on `cmd-develop`.
2) Add some datasets / dataset versions to a collection's JSON manually or via the UI, for example this snippet should work:
```
"datasets": [{
        "id": "931a8a2a-0dc8-42b6-a884-7b6054ed3b68",
        "title": "Acceptance test",
        "state": "InProgress",
        "uri": "http://localhost:22000/datasets/931a8a2a-0dc8-42b6-a884-7b6054ed3b68"
    }],
    "datasetVersions": [{
        "id": "931a8a2a-0dc8-42b6-a884-7b6054ed3b68",
        "title": "Acceptance test",
        "edition": "time-series",
        "version": "1",
        "uri": "http://localhost:22000/datasets/931a8a2a-0dc8-42b6-a884-7b6054ed3b68/editions/time-series/versions/1",
        "state": "InProgress"
    }],
```
3) Change the 'state' of those datasets / dataset versions and check that it moves into the correct column.
4) Approve and delete collection buttons still show at the correct times.


### Who can review

Anyone but me.
